### PR TITLE
Add config, packet_router protos

### PR DESCRIPTION
**Pull request by dpezely**
This supersedes https://github.com/helium/proto/pull/157 which was too convoluted due to many `merge master`, etc.

Add:
- `package helium.config`
- `package helium.packet_router`

### CODE REVIEW

**Review by michaeldjeffrey — APPROVED** [review]

File: src/service/packet_router.proto [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  data_rate datarate = 3;
+}
+
+message packet_router_packet_down_v1 {
+  bytes payload = 1;
+  window_v1 rx1 = 2;
+  window_v1 rx2 = 3;
+}
+
+service gateway {
```
Comment (madninja) [inline src/service/packet_router.proto:51]: why is this called `gateway`? it's the `packet_router` service no?

Comment (michaeldjeffrey) [reply src/service/packet_router.proto:51]: I remember that it's gone through quite a few name changes. 
If I'm reaching for plausible reasons how we arrived here...

In the Packet Router namespace, this is the service that Gateways use.

Comment (macpie) [reply src/service/packet_router.proto:51]: yes I think that the only real reason

Comment (macpie) [reply src/service/packet_router.proto]: We changed it to `packet` as this is what the service is moving around

**Review by madninja — APPROVED** [review]

**Pull request merged**